spec/fixtures/error_corpus/javascript_errors.txt

```diff
@@ -53,13 +53,13 @@ if ({a: 'b'} {c: 'd'}) {
     (object (pair (identifier) (string)))
     (ERROR (object (pair (identifier) (string))))
     (statement_block
-      (ERROR (function
+      (function
         (formal_parameters (identifier))
         (statement_block (expression_statement (identifier)))))
       (expression_statement
         (function
           (formal_parameters (identifier))
-          (statement_block (expression_statement (identifier))))))))
+          (statement_block (expression_statement (identifier)))))))
```

Comment:
@maxbrunsfeld I saw your commit on master that updates this test 👍 🙇 

I'm curious though why the build for this branch is causing 119 build failures? From what I can see, it looks like this build should be fine, but I'm missing something.